qt_ui/windows/newgame/WizardPages/QFactionSelection.py

```diff
@@ -228,6 +321,22 @@ def updateFactionUnits(self, units: Union[set, list]):
         for d in deletes:
             units.remove(d)
 
+    def _format(self, cb: QComboBox, callback: Callable):
+        add_button = QPushButton("+")
+        add_button.setStyleSheet("QPushButton{ font-weight: bold; }")
+        add_button.setFixedWidth(50)
+        add_button.clicked.connect(callback)
+        hbox = QHBoxLayout()
+        hbox.addWidget(cb)
+        hbox.addWidget(add_button)
+        if cb.count() == 0:
+            cb.hide()
+            add_button.hide()
+        else:
+            cb.show()
+            add_button.show()
+        return hbox
```

Comment:
These are causing the popups, changing them to `cb.setEnabled(False)` or `cb.setEnabled(True)` (and for `add_button` respectively) should fix the issue. If you really want to hide them, you'll need to use `setStyleSheet` and use CSS instead ;)